apps/vanilla/src/pages/blackjack/script/displayers.ts

```diff
@@ -0,0 +1,113 @@
+import { isButtonElement } from '@js-camp/vanilla/type-guard/button';
+
+import { isHTMLElement } from '@js-camp/vanilla/type-guard/element';
+
+import { Subscriber } from '../models';
+
+/** Represent the data for the displayers to render. */
+export type ResultData = {
+
+	/** The array of dice values. */
+	dicesOrder: number[];
+
+	/** The total value of all the dices. */
+	totalScore: number;
+};
```

Comment:
We can add `readonly` our type. Please check other types as well
```suggestion
export type ResultData = {

	/** The array of dice values. */
	readonly dicesOrder: number[];

	/** The total value of all the dices. */
	readonly totalScore: number;
};
```